sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/FileLastWrittenMode.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.file.share.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/** Defines values for FileLastWrittenMode. */
+public enum FileLastWrittenMode {
```

Comment:
Does this refere to `ShareFileItemProperties.getLastWrittenOn` ? Should this be called `FileLastWrittenOnMode` ?

Reply:
I think this the name we agreed on in the arch board session?